contracts/GSN/GSNRecipient.sol

```diff
@@ -11,17 +11,30 @@ import "./IRelayHub.sol";
  * must do so themselves.
  */
 contract GSNRecipient is IRelayRecipient, GSNContext, GSNBouncerBase {
+    /**
+     * @dev Returns the RelayHub address for this recipient contract.
+     */
     function getHubAddr() public view returns (address) {
         return _relayHub;
     }
 
+    /**
+     * @dev This function returns the version string of the RelayHub for which
+     * this recipient implementation was built. It's not currently used, but
+     * may be used by tooling.
+     */
```

Comment:
Should we remove the two lines of comments below if we are adding this?

Reply:
I left them in because I want the comment to be there for the future but didn't want it to be in the docs.